### DISCUSSION TITLE
How to restart invidious on a cron?

### DISCUSSION THREAD

**Discussion by Incognify** [tmiland/invidious-updater#62] [Q&A] [answered]
Disregard. I'm an idiot. I had a typo in the cron.

**Comment by tmiland** [comment, marked as answer]
Hey,

I'm using this for restarting Invidious:

```shell
* * * * * curl -o /dev/null -s -m 10 http://169.254.169.254/metadata/v1/interfaces/private/0/ipv4/address:3000 || systemctl restart invidious
* * * * * sleep 30 && curl -o /dev/null -s -m 10 http://169.254.169.254/metadata/v1/interfaces/private/0/ipv4/address:3000 || systemctl restart invidious
@hourly systemctl restart invidious
```

This is for a DigitalOcean droplet, where ``` http://169.254.169.254/metadata/v1/interfaces/private/0/ipv4/address:3000 ```  are for the private IP address. The second one with a 30 sec. delay, so it'll run every minute, or each every 30 seconds.

So if curl is failing on ip:port, it'll restart the service. 

And one for hourly restart just to be sure.
